### PR TITLE
WW-162 | disabling tests for NCF dialog

### PR DESCRIPTION
**Pull request by rybmat**
Tickets: 
* https://wikia-inc.atlassian.net/browse/WW-162
* https://wikia-inc.atlassian.net/browse/WW-124

NCF dialog is disabled so we are disabling its tests.

@Wikia/west-wing @nikodamn

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/communitypagetests/CommunityPageSalesPitchDialogTests.java [hunk 1]
```diff
@@ -3,13 +3,15 @@
 import com.wikia.webdriver.common.contentpatterns.PageContent;
 import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.RelatedIssue;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/communitypagetests/CommunityPageSalesPitchDialogTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by nikodamn** [comment]
LGTM

**Comment by SebastianMarzjan** [comment]
LGTM

**Pull request merged**